### ISSUE TITLE
Overlapping tab issue with multiple tabs

### ISSUE THREAD

**Issue by orschiro** [piqus/sublime-elementary#2]
Dear @piqus, 

Thanks very much for maintaining this excellent theme!

I encounter a small problem with overlapping tabs. 

Can you or someone else confirm this?

Thankfully,

Robert

![sublimeeosissuetabs](https://cloud.githubusercontent.com/assets/170947/7809910/129a47ce-039e-11e5-9f37-d1efed0b5965.png)

**Comment by piqus** [comment]
Yup, that's true :crying_cat_face:  Tabs overlaps badly other tabs in both variants.
